scripts/screens/all_screens.py

```diff
@@ -7,7 +7,7 @@
 from .ClanScreen import ClanScreen
 from .ClanSettingsScreen import ClanSettingsScreen
 from .ClearingScreen import ClearingScreen
-from .EventEdit import EventEdit
+from .EventEditScreen import EventEdit
```

Comment:
```suggestion
from .EventEditScreen import EventEditScreen
```
can we get the class renamed too? 🥺
(don't do it from my suggestion, do it through IDE lol)

Reply:
As you command o7